src/Test.FunctionalTests.BinSkim.Driver/BaselineTestData/Expected/Binskim.linux-x64.dll.sarif

```diff
@@ -210,48 +209,19 @@
                 "equivalentBinScopeRuleReadableName": "WXCheck"
               }
             }
-          ],
-          "properties": {
-            "comments": "A security and correctness analyzer for portable executable and MSIL formats."
-          }
+          ]
         }
       },
       "invocations": [
         {
-          "toolConfigurationNotifications": [
-            {
-              "locations": [
-                {
-                  "physicalLocation": {
-                    "artifactLocation": {
-                      "uri": "file:///Z:/src/Test.FunctionalTests.BinSkim.Driver/BaselineTestData/Binskim.linux-x64.dll",
-                      "index": 0
-                    }
-                  }
-                }
-              ],
-              "message": {
-                "text": "'Binskim.linux-x64.dll' was not evaluated because its PDB could not be loaded (E_PDB_FORMAT)."
-              },
-              "level": "error",
-              "descriptor": {
-                "id": "ERR997.ExceptionLoadingPdb"
-              }
-            }
-          ],
-          "executionSuccessful": false
+          "executionSuccessful": true
```

Comment:
>executionSuccessful [](http://example.com/codeflow?start=11&length=19)

For this file, our broken build system did not copy supporting PDB libraries properly and so this scan session raised an exception. And we checked in errors that indicate the scan literally failed. We should try to prevent this from happening again, I filed a work item noted in the description. It's hard to read JSON diffs and understand what changes are good or not...